src/index.html

```diff
@@ -11,12 +11,68 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="http://mateacademy.com"
+        class="logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="list-item">
+            <a class="item-link is-active">
```

Comment:
The `href` attribute is missing a URL. Consider using a meaningful URL or a JavaScript function to handle the click event for better accessibility and usability.